src/gwt/src/org/rstudio/studio/client/workbench/views/environment/view/MemoryUsageSummary.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * MemoryUsageSummary.java
+ *
+ * Copyright (C) 2021 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.studio.client.workbench.views.environment.view;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.Document;
+import com.google.gwt.dom.client.Element;
+import com.google.gwt.dom.client.TableCellElement;
+import com.google.gwt.dom.client.TableRowElement;
+import com.google.gwt.resources.client.CssResource;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HTMLPanel;
+import com.google.gwt.user.client.ui.Label;
+import com.google.gwt.user.client.ui.Widget;
+import org.rstudio.core.client.ElementIds;
+import org.rstudio.core.client.StringUtil;
+import org.rstudio.core.client.widget.MiniPieWidget;
+import org.rstudio.studio.client.workbench.views.environment.model.MemoryStat;
+import org.rstudio.studio.client.workbench.views.environment.model.MemoryUsage;
+import org.rstudio.studio.client.workbench.views.environment.model.MemoryUsageReport;
+
+public class MemoryUsageSummary extends Composite
+{
+   interface Style extends CssResource
+   {
+      String kbCell();
+      String stats();
+   }
+
+   /**
+    * Creates a new memory usage summary widget
+    *
+    * @param report The memory usage report to show in the widget.
+    */
+   public MemoryUsageSummary(MemoryUsageReport report)
+   {
+      report_ = report;
+
+      initWidget(GWT.<MemoryUsageSummary.Binder>create(MemoryUsageSummary.Binder.class).createAndBindUi(this));
+
+      MemoryUsage usage = report.getSystemUsage();
+
+      // Create the pie chart showing the percentage of memory used
+      int percent = usage.getPercentUsed();
+      pie_.setForeColor(usage.getColorCode());
+      pie_.setBackColor(MemUsageWidget.MEMORY_PIE_UNUSED_COLOR);
+      pie_.setPercent(percent);
+      pie_.getElement().getStyle().setWidth(150, com.google.gwt.dom.client.Style.Unit.PX);
+      pie_.getElement().getStyle().setHeight(150, com.google.gwt.dom.client.Style.Unit.PX);
+      pie_.setTitle("Memory in use: " + usage.getPercentUsed());
+      pie_.setDescription("Pie chart depicting the percentage of total memory in use");
+      ElementIds.assignElementId(pie_, ElementIds.MEMORY_PIE_FULL);
+      pieLabel_.setText(percent + "%");
+
+      Element statsTable = Document.get().createTableElement();
```

Comment:
This table has no accessible name (no programmatic association with the "Memory Usage" pseudo-header above it).

Variety of ways to fix this:
- include a caption element (i.e. `<table><caption>Memory Usage</caption>...`)
- assign unique ID to the existing "Memory Usage" div (and possibly make that an actual `<h1>` while you're at it), then reference it from the table (i.e. `<table aria-labelledby="the unique id">`)
- use aria-label: `<table aria-label="Memory Usage">`

Reply:
Duplicating text is the worst. I've done #2.